packages/dapp/pages/index.tsx

```diff
@@ -14,11 +15,23 @@ const index: FC = (): JSX.Element => {
   const managedContracts = useManagerManaged();
 
   useEffectAsync(async () => {
-    if (managedContracts) {
-      setTwapPrice(await managedContracts.dollarTwapOracle.consult(managedContracts.dollarToken.address));
```

Comment:
this line it's doing nothing, this need refactor along with the hooks at `use-manager-managed.tsx` subtask here

Reply:
I don't quite understand but if it's a big project this can be a new bounty if you can clarify what exactly should be done!